base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -138,7 +138,7 @@ impl ConsensusConstants {
 
     #[allow(clippy::identity_op)]
     pub fn rincewind() -> Self {
-        let target_block_interval = 60;
+        let target_block_interval = 120;
         let difficulty_block_window = 150;
```

Comment:
```suggestion
        let difficulty_block_window = 90;
```
You can make the block window smaller if you make the interval bigger

60 -> 150
120 -> 90